src/Translation_unit.ml

```diff
@@ -79,7 +79,8 @@ let parse_print (XUnit xunit) (conf: Conf.t) ~input_name:iname
     in
     if Conf.debug then
       dump xunit dir base ".old" ".ast" (xunit.normalize (ast, cmts)) ;
-    xunit.init_cmts source ast cmts ;
+    Source.init source ;
+    xunit.init_cmts ast cmts ;
```

Comment:
It would be good if the implicit dependence on the order of initialization was either hidden (can Cmts call Source.init) or at least resulted in a reasonable exception if gotten wrong.